change/@fluentui-eslint-plugin-8b1b38b2-2c2b-4b25-b911-1db1b1ead64d.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Create no-context-default-value lint rule",
+  "packageName": "@fluentui/eslint-plugin",
+  "email": "bernardo.sunderhus@gmail.com",
+  "dependentChangeType": "patch"
```

Change this to `none` to prevent unnecessary bumps 
```suggestion
  "dependentChangeType": "none"
```